bin/export_co.py

```diff
@@ -110,7 +111,10 @@
         co = hh[1]['CO'][:]
         hh.close()
     else:
-        co = sp.eye(dd.size)
+        w = rr>0.
+        co = sp.zeros_like(dd)
```

Comment:
`zero_like` is much less efficient than `zero(dd.shape)` and `dd*0`, please consider one of the latter options

Reply:
done